### PR TITLE
Grunt 0.4 compatible implementation

### PR DESCRIPTION
**Pull request by sleeper**
Hi there,

A Grunt 0.4 compatible implementation based on [mklabs/tiny-lr](https://github.com/mklabs/tiny-lr).

### CODE REVIEW

**Comment by tkellen** [comment]
@sindresorhus Are you still maintaining this repo?

**Comment by sindresorhus** [comment]
@tkellen Frederick stepped in an did this. We could never get anywhere with the Livereload author.

This looks good too me. Could you review and merge?

**Comment by MoOx** [comment]
There are some issues with this branch, be careful if you want to merge (+doc seems out of date)

**Comment by sindresorhus** [comment]
@MoOx care to elaborate?

**Comment by MoOx** [comment]
I just cannot make this work. livereload-start task tell me the server is started, but the livereload command (trigger) tell me to start the server. Tried with latest version of grunt (grunt-cli v0.1.6 & grunt v0.4.0rc5) & latest version of grunt-contrib-watch.
The doc mentions a [grunt-regarde](https://github.com/sleeper/grunt-regarde) package (grunt-contrib-watch replacement, but it seems deleted by @sleeper)

File: README.md [hunk 1]
```diff
@@ -13,48 +12,93 @@ npm install grunt-contrib-livereload --save-dev
 [grunt]: http://gruntjs.com/
 [Getting Started]: https://github.com/gruntjs/grunt/blob/devel/docs/getting_started.md
 
-
 ## Livereload task
 _Run this task with the `grunt livereload` command._
```
Comment (sindresorhus) [inline README.md]: @sleeper Mention that the user must use the `regarde` task instead of `watch`. (with link)

**Comment by sindresorhus** [comment]
@MoOx You can find the regarde task here: https://github.com/yeoman/grunt-regarde

The regarde task was created because of shortcomings in the watch task which doesn't give access to changed files because it spawns tasks in subprocesses.

@tkellen I've tested this with latest grunt and it works ok.

**Comment by MoOx** [comment]
@sindresorhus Ok great. Thanks for the notice. I'll try that right now.

**Comment by tkellen** [comment]
@sindresorhus Sorry, I don't use LR, and I can't add any other plugins to my project load at the moment.  If there are any contrib projects you don't feel you have the time to maintain, we should discuss finding another team member to steward them.  Hit me up on IRC any time!

**Comment by sleeper** [comment]
Hi Guys,

Sorry to step in so late in the discussion, but I was stuck in meetings.
I'll have a look at that tonight or tomorrow morning early.
Sorry for the inconvenience.

**Comment by sindresorhus** [comment]
@tkellen Didn't imply you should. I'd be happy to maintain this if @sleeper isn't interested (@sleeper are you?).

File: README.md (hunk 1 above)
Comment (sleeper) [inline README.md]: Ok, done.

**Comment by sleeper** [comment]
Yeah, I'm interested ;)

**Comment by sleeper** [comment]
OK: I've updated the README (and took into account the fact that it's generated), supposedly fix the fact that travis was failing testing it, and rebased & squashed the commit on master

**Comment by MoOx** [comment]
This work for me now. But is there anything to do for CSS live reload (without entire page reload) ? It reloads entire page for me (I use compass) ?

**Comment by MoOx** [comment]
Ok it's simple. Just use a different watch(regarde) on scss (task: compass) and css (task: livereload).
So this PR fully work for me ! Thanks you guys.

File: package.json [hunk 2]
```diff
@@ -25,16 +25,19 @@
     "node": ">= 0.8.0"
   },
   "scripts": {
-    "test": "grunt test"
+    "test": "node node_modules/mocha/bin/mocha test/*_test.js"
```
Comment (sindresorhus) [inline package.json]: Why are you including grunt-simple-mocha and then using mocha directly?

Comment (sleeper) [inline package.json]: It was before I installed grunt-simple-mocha ... Now `grunt test` is working correctly with mocha .. I'll revert properly ;)

**Comment by sindresorhus** [comment]
Landed. Thanks @sleeper :)

**Comment by sleeper** [comment]
Yes !!! Many thanks !! :)

**Pull request closed without merging**